src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,179 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    private static final String NAME = "duke.txt";
+    private static final String DIRECTORY = "data";
+
+    public static void main(String[] args) throws IOException {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int taskCount = 0;
+
+        printGreeting();
+        FileManager file = new FileManager(NAME, DIRECTORY);
+        try {
+            taskCount = file.loadSavedFile(list, taskCount);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        String command = sc.nextLine();
+
+        while (!command.equals("bye")) {
+            String taskCategory = extractTaskCategory(command);
+            try {
+                switch (taskCategory) {
+                case "list":
```

Comment:
It might be better if the strings "list", "done" etc. be constants

Reply:
Thanks! Will be changing in my upcoming version